src/Context/LoginContext.php

```diff
@@ -80,6 +81,45 @@ public function stepIAmNotLoggedIn()
      * @param string $password
      */
     public function stepILogInWith($email, $password)
+    {
+        $this->loginWith($email, $password);
+
+        // Check if MFA module is installed
+        if (!class_exists(RegisteredMethod::class)) {
+            return;
+        }
+
+        // Skip MFA registration if MFA module installed
+        $this->getMainContext()->getSession()->wait(300);
+        $page = $this->getMainContext()->getSession()->getPage();
+        $mfa = $page->find('css', '#mfa-app');
+        if (!$mfa) {
+            return;
+        }
+        $btns = $page->findAll('css', '.mfa-action-list__item .btn');
```

Comment:
Is [`stepIPressTheButton()`](https://github.com/silverstripe/silverstripe-behat-extension/blob/ba2e93131e39774d51d87292d9883f5f222a8bdf/src/Context/BasicContext.php#L438) not available in this context?

Reply:
Nope, LoginContext (this file) implements Context.  It would need to extend BasicContext for this to be available.

$this->getMainContext() will return the current FeatureContext, which will vary depending on the module.